rest-api-spec/src/main/resources/rest-api-spec/api/text_structure.find_message_structure.json

```diff
@@ -0,0 +1,98 @@
+{
+  "text_structure.find_message_structure":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/find-message-structure.html",
+      "description":"Finds the structure of a list of messages. The messages must contain data that is suitable to be ingested into Elasticsearch."
+    },
+    "stability":"stable",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_text_structure/find_message_structure",
+          "methods":["GET", "POST"]
+        }
+      ]
+    },
+    "params":{
+      "lines_to_sample":{
+        "type":"int",
+        "description":"How many lines of the file should be included in the analysis",
+        "default":1000
+      },
```

Comment:
For this endpoint, presumably the caller is setting this implicitly by how many messages they choose to supply in the input array. So this number doesn't need to be a separate parameter.